Cargo.lock

```diff
@@ -5040,9 +5040,9 @@ dependencies = [
 
 [[package]]
 name = "textwrap"
-version = "0.14.2"
+version = "0.15.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "0066c8d12af8b5acd21e00547c3797fde4e8677254a7ee429176ccebbe93dd80"
+checksum = "b1141d4d61095b28419e22cb0bbf02755f5e54e0526f97f1e3d1d160e60885fb"
```

Comment:
These changes look unrelated?

Reply:
Not sure, thought I had to include the changes to `Cargo.lock`. I've had problems in the past by not including changes to that file. What am I supposed to do here?

Reply:
Since this PR doesn't appear to update any Cargo.toml files, you should be able to drop these changes by resetting from master; e.g. `git restore --source=origin/master -- Cargo.lock`.